packages/block-library/src/video/edit.js

```diff
@@ -136,13 +138,17 @@ function VideoEdit( {
 	const placeholder = ( content ) => {
 		return (
 			<Placeholder
-				className="block-editor-media-placeholder"
+				className={ clsx( 'block-editor-media-placeholder', {
+					[ borderProps.className ]:
+						!! borderProps.className && ! isSingleSelected,
+				} ) }
 				withIllustration={ ! isSingleSelected }
 				icon={ icon }
 				label={ __( 'Video' ) }
 				instructions={ __(
 					'Upload a video file, pick one from your media library, or add one with a URL.'
 				) }
+				style={ ! isSingleSelected ? { ...borderProps.style } : {} }
```

Comment:
```suggestion
				style={ ! isSingleSelected ? borderProps.style : {} }
```

It looks like the recent updates [reintroduced](https://github.com/WordPress/gutenberg/pull/63777#discussion_r1713251390) the unnecessary destructuring of the border styles object.

Reply:
Sorry, missed to remove it.